### ISSUE TITLE
Init flow requires an empty directory

### ISSUE THREAD

**Issue by divyenduz** [prisma/prisma2#344]
The following workflow breaks: Setting up a new project with local CLI. 

1. Setup a new project, so, yarn init -y
2. yarn add -D prisma2
3. yarn prisma2 init

```
$ [p2-internal]$ ls
node_modules    package.json    yarn.lock
$ [p2-internal]$ yarn prisma2 init
yarn run v1.16.0
warning package.json: No license field
$ /Users/divyendusingh/Documents/prisma/p2-internal/node_modules/.bin/prisma2 init
Error: Error: Can't start prisma2 init as the files node_modules, package.json, yarn.lock exist in ./
Please either run prisma2 init in an empty directory
or provide a directory to initialize in: prisma2 init sub-dir
```

This forces me to install `prisma2` globally for using `init`. 

For now, this is as per the spec: https://prisma-specs.netlify.com/cli/init/

**Comment by nikolasburk** [comment]
Idea how this can be optimized:

```
$ prisma2 init
```

- Case 1: **User selects example at the end of `init` flow** 
    - if **current directory is empty**, download example into current directory
    - if **current directory is _not_ empty**, download the entire example project directory into the current directory and name it after the example (e.g. `graphql`, `graphql-auth`, `grpc`, `rest-express`,...)
- Case 2: **User does _not_ select example at the end of `init` flow** 
    - if there's a **naming conflict** (e.g. a `prisma` directory already exists in current directory), fail with error message that points out the naming conflict
    - if there's **no naming conflict**, all good :)

**Comment by timsuchanek** [comment]
We decided to support this in the new init flow https://prisma-specs.netlify.com/cli/init/

**Comment by divyenduz** [comment]
This is fixed in `prisma2@2.0.0-alpha.150`, please re-open a new issue if the problem persists for you.
